requirements2.txt

```diff
@@ -0,0 +1,64 @@
+argon2-cffi==20.1.0
+async-generator==1.10
+attrs==20.3.0
+backcall==0.2.0
+bleach==3.3.0
+cffi==1.14.5
+colorama==0.4.4
+cycler==0.10.0
+decorator==4.4.2
+defusedxml==0.7.1
+entrypoints==0.3
+ipykernel==5.5.0
+ipython==7.21.0
+ipython-genutils==0.2.0
+jedi==0.18.0
+Jinja2==2.11.3
+jsonschema==3.2.0
+jupyter-client==6.1.12
+jupyter-contrib-core==0.3.3
+jupyter-contrib-nbextensions==0.5.1
+jupyter-core==4.7.1
+jupyter-highlight-selected-word==0.2.0
+jupyter-latex-envs==1.4.6
+jupyter-nbextensions-configurator==0.4.1
+jupyterlab-pygments==0.1.2
+kiwisolver==1.3.1
+lxml==4.6.2
+MarkupSafe==1.1.1
+matplotlib==3.3.4
+mistune==0.8.4
+nbclient==0.5.3
+nbconvert==6.0.7
+nbformat==5.1.2
+nest-asyncio==1.5.1
+notebook==6.2.0
+numpy==1.20.1
+packaging==20.9
+pandas==1.2.3
+pandocfilters==1.4.3
+parso==0.8.1
+pickleshare==0.7.5
+Pillow==8.1.2
+prometheus-client==0.9.0
+prompt-toolkit==3.0.17
+pycparser==2.20
+Pygments==2.8.1
+pyparsing==2.4.7
+pyrsistent==0.17.3
+python-dateutil==2.8.1
+pytz==2021.1
+pywin32==300
```

Comment:
Everything worked fine on my machine using the new ```requirements2.txt``` with the exception of ```pywin32==300``` which I had to remove. Is this a dependency that you need?

Reply:
Just dropped `pywin32==300` on https://github.com/danieljbae/CS-4502-Project/pull/10/commits/399d3b6b665c88214dfa83df33e606dbdcfc166f